riotfile.py

```diff
@@ -476,6 +476,29 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
             pkgs={
                 "vertica-python": [">=0.6.0,<0.7.0", ">=0.7.0,<0.8.0"],
             },
+            # venvs=[
+            # FIXME: tests fail on vertica 1.x
```

Comment:
do we need these before we can merge?

Reply:
Same for the other FIXMEs ?

Reply:
The FIXMEs are all what we **should** be testing (was missing test coverage before), but we currently fail them as we don't have support for vertica/sqlalchemy/algoliasearch/... latest versions. 
I'm okay with merging this, I left these comments in to easily add testing once we add support in the future. I thought adding actual support was outside the scope of this PR.

Reply:
> (was missing test coverage before)

ok, got it, as long as we aren't removing test coverage for something and it is expected but unsupported 👍🏻